setup.py

```diff
@@ -0,0 +1,32 @@
+import subprocess
+
+from setuptools import find_packages, setup
+
+
+def run_code_formatters():
+    for tool in ["isort .", "black .", "flake8 src/", "flake8 test/", "mdformat ."]:
+        print(f"running `{tool}`")
+        subprocess.run(tool, shell=True)
+
+
+if __name__ == "__main__":
+    setup(
+        name="cs101",
+        version="1.0",
+        url="https://github.com/jameshughes89/cs101",
+        python_requires=">=3.8",
+        packages=find_packages(),
+        install_requires=[
```

Comment:
Should probably set up dependabot to help manage these for you as well. You should be able to rip it wholesale off of [duckbot](https://github.com/duck-dynasty/duckbot/blob/main/.github/dependabot.yml).

Reply:
I'll make a new PR for that